js_modules/dagit/packages/core/src/launchpad/LaunchButton.tsx

```diff
@@ -173,6 +175,7 @@ export const LaunchButtonDropdown = ({
 
 interface ButtonWithConfigurationProps {
   title: string;
+  warning?: string;
```

Comment:
Maybe a `React.ReactNode` for this?

Reply:
Ahh yeah that should work and would be nice futureproofing 🙌